addons/mail/static/src/views/activity/activity_controller.js

```diff
@@ -52,7 +52,7 @@ export class ActivityController extends Component {
                     model: this.props.resModel,
                 });
                 await messaging.openActivityForm({ thread });
-                this.model.load();
+                this.model.load({ "domain": this.model.originalDomain });
```

Comment:
Very important change
```suggestion
                this.model.load({ domain: this.model.originalDomain });
```